packages/odyssey-react-mui/src/labs/OdysseyPickers/PickerWithOptionAdornment.tsx

```diff
@@ -45,16 +45,21 @@ import { Heading6 } from "../../Typography.js";
 import { Tag } from "../../Tag.js";
 
 type Adornment = ReactNode | string;
+type RightAdornment = {
+  content: ReactNode | string;
```

Comment:
```suggestion
  content: ReactNode | string;
```

This can simply be `ReactNode`. It includes `string`.